geomesa-core/src/test/scala/org/locationtech/geomesa/core/index/IndexValueEncoderTest.scala

```diff
@@ -276,10 +276,11 @@ class IndexValueEncoderTest extends Specification {
   // 2.  WKB-encoded geometry
   // 3.  start-date/time
   def _encodeIndexValue(entry: SimpleFeature): Value = {
-    import org.locationtech.geomesa.core.index.IndexEntry._
-    val encodedId = entry.sid.getBytes
-    val encodedGeom = WKBUtils.write(entry.geometry)
-    val encodedDtg = entry.dt.map(dtg => ByteBuffer.allocate(8).putLong(dtg.getMillis).array()).getOrElse(Array[Byte]())
+    val encodedId = entry.getID.getBytes
+    val encodedGeom = WKBUtils.write(entry.getDefaultGeometry.asInstanceOf[Geometry])
```

Comment:
See comment below - this is an old method copied here to test back compatibility.